core/src/components/input-password-toggle/input-password-toggle.ionic.scss

```
@@ -3,10 +3,25 @@
 
 // Ionic Input Toggle Password
 // --------------------------------------------------
-:host ion-button.button-has-icon-only {
-  --padding-end: inherit;
-  --color: #{globals.$ionic-color-neutral-500};
+:host ion-button.in-input-password-toggle {
```

```suggestion
:host ion-button {
```

Do we need this class? The styles seem to apply fine with the above